Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -33,7 +33,8 @@ extension TelemetryWrapperProtocol {
 class TelemetryWrapper: TelemetryWrapperProtocol {
     static let shared = TelemetryWrapper()
     let legacyTelemetry = Telemetry.default
-    let glean = Glean.shared
+    // Ecosia: let glean = Glean.shared
```

Comment:
I noticed you sometimes used `// Ecosia: remove Glean dependency //` and other times `// Ecosia:`(less often). Just wondering, is there any intentional pattern? Since the main intention is making our lives easier when merging FF changes, I guess all approaches achieve it anyway 😅

Reply:
Yeah, to be honest, the intention is to have `// Ecosia: [explanation] // [commented_code]`. Throuout the work done and testing, I de-prioritized (then forgot) to apply it 😇 . Whilst both achieve pretty much the same end goal (i.e. marking Ecosia's changes), I'd love to add another commit and align all changes with the `[explanation]` bit.
Thanks for highlight it!